src/api/data-model/Authentication.ts

```diff
@@ -0,0 +1,6 @@
+export interface Authentication {
+  /**
+   * The token used to authenticate the solver
+   */
+  token: string;
+}
```

Comment:
Because it might not be a token in the future but some other authentication method. I envisioned this to reflect the AuthenticationOptions type. When more boolean options like supportsToken are added, their result will be stored in the Authentication type.